<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="1" column="1">
     <widget class="QPushButton" name="encodePushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Encode</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="cancelPushButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QScrollArea" name="scrollArea">
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>649</width>
         <height>774</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Processing</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="title">
              <string>Input/Output</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="2" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Output File:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QPushButton" name="inputFileBrowsePushButton">
                <property name="text">
                 <string>Browse...</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QPushButton" name="outputFileBrowsePushButton">
                <property name="text">
                 <string>Browse...</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="inputFileLineEdit"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Input File:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="outputFileLineEdit"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_6">
             <property name="title">
              <string>Trimming</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="1" column="0">
               <widget class="QRadioButton" name="trimNoneRadioButton">
                <property name="text">
                 <string>No Trimming</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QRadioButton" name="trimDurationRadioButton">
                <property name="text">
                 <string>Duration</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QRadioButton" name="trimStartEndRadioButton">
                <property name="text">
                 <string>Start/End Times</string>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="QTimeEdit" name="trimStartEndEndTimeEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="displayFormat">
                 <string>hh:mm:ss.zzz</string>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QTimeEdit" name="trimStartEndStartTimeEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="displayFormat">
                 <string>hh:mm:ss.zzz</string>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QTimeEdit" name="trimDurationStartTimeEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="displayFormat">
                 <string>hh:mm:ss.zzz</string>
                </property>
               </widget>
              </item>
              <item row="3" column="3">
               <widget class="QTimeEdit" name="trimDurationDurationTimeEdit">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="displayFormat">
                 <string>hh:mm:ss.zzz</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>Start Time:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>End Time/Duration:</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_7">
             <property name="title">
              <string>Resizing</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="3" column="1">
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>Left (px):</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QCheckBox" name="resizeCheckBox">
                <property name="text">
                 <string>Resize</string>
                </property>
               </widget>
              </item>
              <item row="6" column="2">
               <widget class="QSpinBox" name="cropBottomSpinBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="4" column="2">
               <widget class="QSpinBox" name="cropRightSpinBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QSpinBox" name="cropLeftSpinBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QSpinBox" name="cropTopSpinBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="3" column="2">
               <widget class="QLabel" name="label_16">
                <property name="text">
                 <string>Right (px):</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QCheckBox" name="cropCheckBox">
                <property name="text">
                 <string>Crop</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_13">
                <property name="text">
                 <string>Width (px):</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Height (px):</string>
                </property>
               </widget>
              </item>
              <item row="5" column="2">
               <widget class="QLabel" name="label_18">
                <property name="text">
                 <string>Bottom (px):</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QLabel" name="label_17">
                <property name="text">
                 <string>Top (px):</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QSpinBox" name="resizeWidthSpinBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="2">
               <widget class="QSpinBox" name="resizeHeightSpinBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
             <zorder>groupBox_6</zorder>
             <zorder>label_13</zorder>
             <zorder>label_14</zorder>
             <zorder>resizeWidthSpinBox</zorder>
             <zorder>resizeHeightSpinBox</zorder>
             <zorder>cropCheckBox</zorder>
             <zorder>resizeCheckBox</zorder>
             <zorder>label_15</zorder>
             <zorder>label_16</zorder>
             <zorder>label_17</zorder>
             <zorder>label_18</zorder>
             <zorder>cropLeftSpinBox</zorder>
             <zorder>cropRightSpinBox</zorder>
             <zorder>cropTopSpinBox</zorder>
             <zorder>cropBottomSpinBox</zorder>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Advanced</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>Rate Control</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="1" column="2">
               <widget class="QDoubleSpinBox" name="rateCRFSpinBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="decimals">
                 <number>1</number>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>CRF:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Mode:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QComboBox" name="rateModeComboBox">
                <property name="sizeAdjustPolicy">
                 <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
                </property>
                <property name="minimumContentsLength">
                 <number>0</number>
                </property>
                <item>
                 <property name="text">
                  <string>VBR</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>CBR</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>CRF</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_19">
                <property name="text">
                 <string>Target Mode:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="rateTargetModeComboBox">
                <item>
                 <property name="text">
                  <string>Bit Rate</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>File Size</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Target File Size (kB):</string>
                </property>
               </widget>
              </item>
              <item row="0" column="4">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Target Bit Rate (kB/s):</string>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <widget class="QSpinBox" name="rateTargetFileSizeSpinBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="4">
               <widget class="QSpinBox" name="rateTargetBitRateSpinBox"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>Streams</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="1" column="1">
               <widget class="QComboBox" name="streamAudioComboBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QComboBox" name="streamVideoComboBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QComboBox" name="streamSubtitlesComboBox">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Video:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Audio:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="2">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Subtitles:</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_8">
             <property name="title">
              <string>Custom</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>Custom Encoder Parameters:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLineEdit" name="customEncoderParametersLineEdit"/>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>682</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
